### DISCUSSION TITLE
Refer to CacheConfiguration.Default in new secret store extensions

### DISCUSSION THREAD

**Discussion by tomkerkhove** [arcus-azure/arcus.security#271] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Providing a cache configuration is mandatory nowadays, but people are not always aware of `CacheConfiguration.Default`.

**Describe the solution you'd like**
Update the XML docs to mention `CacheConfiguration.Default` in new secret store extensions

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by fgheysels** [comment]
https://github.com/arcus-azure/arcus.security/issues/239#issuecomment-762819792 \o/

**Comment by stijnmoreels** [comment, marked as answer]
Hmm, the obsolete secret store extensions already refer to this one though. Not sure which ones you are referring to?

**Reply by tomkerkhove** [reply]
Ok it is indeed there: https://github.com/arcus-azure/arcus.security/blob/master/src/Arcus.Security.Providers.AzureKeyVault/Extensions/SecretStoreBuilderExtensions.cs#L245

But I might have been looking at a deprecated one: https://github.com/arcus-azure/arcus.security/blob/master/src/Arcus.Security.Providers.AzureKeyVault/Extensions/SecretStoreBuilderExtensions.cs#L179

But that is ok, closing!
